### PR TITLE
Update fields for the Scheduling an Import page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request updates fields for the Scheduling an Import page

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/system/data-schedule-import.html

### CODE REVIEW

File: src/system/data-schedule-import.md [hunk 1]
```diff
@@ -31,16 +31,23 @@ After each scheduled import job, a reindex operation is performed automatically.
 
    - **Entity Type** — Set to one of the following: `Products` or `Customers`
 
+      For the `Advanced Pricing`, `Products`, `Customers and Addresses (single file)`, `Stock Sources` entity’s types are displayed such import behaviors as: Add/Update, Replace, Delete.
+
+      For the `Customer Finances`, `Customers Main File`, `Customers and Addresses` entity’s types are displayed such import behaviors as: Add/Update Complex Data, Delete Entities, Custom Action.
```
Comment (leslietilling) [inline src/system/data-schedule-import.md:36]: This should be a note callout and needs some tweaking on grammar and formatting.

```suggestion
      {:.bs-callout-info}
      For the _Advanced Pricing_, _Products_, _Customers and Addresses (single file)_, and _Stock Sources_ entity types, these additional import behaviors displayed: Add/Update, Replace, and Delete. For the _Customer Finances_, _Customers Main File_, and _Customers and Addresses_ entity types, these additional import behaviors displayed: Add/Update Complex Data, Delete Entities, and Custom Action.
```

File: src/system/data-schedule-import.md [hunk 2]
```diff
@@ -105,7 +112,7 @@ Field | Description
...
+Import Behavior | Defines how complex data is handled if entities, which are being imported, already exist in the database. Complex data for products include categories, websites, custom options, tier prices, related products, up-sells, cross-sells, and associated products data. Complex data for customers include addresses. Options:<br>**Add/Update Complex Data** - The new complex data will be added or updated to the existing complex data for the already existing entries in the database. This is the default value.<br>**Replace** - The existing complex data for the already-existing entities will be replaced.<br>**Delete Entities** - If entities which are being imported already exist in the database, they will be deleted from the database.<br>**Custom Action** - The existing complex of entities will be customized during the import process.
```
Comment (leslietilling) [inline src/system/data-schedule-import.md:115]:
```suggestion
Import Behavior | Defines how complex data is handled if the entities being imported already exist in the database. Complex data for products include categories, websites, custom options, tier prices, related products, up-sells, cross-sells, and associated products data. Complex data for customers include addresses. Options:<br>**Add/Update Complex Data** - The new complex data are added or updated to the existing complex data for existing entries in the database. This is the default value.<br>**Replace** - The existing complex data for the existing entities are replaced.<br>**Delete Entities** - If imported entities already exist in the database, they are deleted from the database.<br>**Custom Action** - The existing complex entities are customized during the import process.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Some adjustments needed.

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**